packages/backend/server/src/models/base.ts

```diff
@@ -1,12 +1,12 @@
-import { Inject, Logger } from '@nestjs/common';
+import { Inject } from '@nestjs/common';
 import { PrismaClient } from '@prisma/client';
 
-import { Config } from '../base';
+import { AFFiNELogger, Config } from '../base';
 import type { Models } from '.';
 import { MODELS_SYMBOL } from './provider';
 
 export class BaseModel {
-  protected readonly logger = new Logger(this.constructor.name);
+  protected readonly logger = new AFFiNELogger(this.constructor.name);
```

Comment:
`AFFiNELogger` can only be injected, otherwise the GCloud implemented `AFFiNELogger` would not work

Reply:
rewrite